spec/support/concerns/graphed_shared_examples.rb

```diff
@@ -95,52 +114,36 @@
   it { is_expected.to callback(:delete_graph_relation).after(:destroy) }
   it { is_expected.to callback(:manage_graph_relation).around(:update) }
 
-  if auto_create
-    it 'should auto_create' do
-      expect(subject).to be
-      expect(graphed_relation).to be
-    end
-    it 'should return the graphed relation of rel_type between from_model.graph_node and to_model.graph_node' do
-      expect(subject).to be
-      expect(from_node.query_as(:from).match("(from)-[r:#{rel_type}]->(to)").where('to.model_id = {m_id}').params(m_id: to_model.id).pluck(:r).count).to eq(1)
-      expect(subject.graph_relation).to be
-      expect(from_node.query_as(:from).match("(from)-[r:#{rel_type}]->(to)").where('to.model_id = {m_id}').params(m_id: to_model.id).pluck(:r).count).to eq(1)
-      graphed_relation = subject.graph_relation
-      expect(graphed_relation.model_id).to eq(subject.id)
-      expect(graphed_relation.model_kind).to eq(subject.kind)
-      expect(graphed_relation.from_node.model_id).to eq(from_model.id)
-      expect(graphed_relation.to_node.model_id).to eq(to_model.id)
-    end
-  else
-    it 'should not auto_create' do
-      expect(subject).to be
-      expect(graphed_relation).not_to be
-    end
+  describe '#graph_model_name' do
+    it { is_expected.to respond_to :graph_model_name }
+    it { expect(subject.graph_model_name).to eq rel_type }
+  end
 
-    it 'should create_graph_relation of rel_type between from_model.graph_node and to_model.graph_node' do
-      expect(subject).to be
-      expect(from_model.graph_node).to be
-      expect(to_model.graph_node).to be
-      expect(from_node.query_as(:from).match("(from)-[r:#{rel_type}]->(to)").where('to.model_id = {m_id}').params(m_id: to_model.id).pluck(:r).count).to eq(0)
-      expect(subject.create_graph_relation).to be
-      expect(from_node.query_as(:from).match("(from)-[r:#{rel_type}]->(to)").where('to.model_id = {m_id}').params(m_id: to_model.id).pluck(:r).count).to eq(1)
-      expect(subject.graph_relation.from_node.model_id).to eq(from_model.id)
-      graphed_relation = subject.graph_relation
-      expect(graphed_relation.model_id).to eq(subject.id)
-      expect(graphed_relation.model_kind).to eq(subject.kind)
-      expect(graphed_relation.from_node.model_id).to eq(from_model.id)
-      expect(graphed_relation.to_node.model_id).to eq(to_model.id)
-    end
+  describe '#graph_model_class' do
+    it { is_expected.to respond_to :graph_model_class }
+    it { expect(subject.graph_model_class).to eq graph_model_class }
+  end
+
+  it 'should auto_create' do
```

Comment:
since none of these actually 'auto_create' it might be better to change this description to something different, or just make it a bare it { ... } block.

Reply:
Actually, they *all* auto create, which is why I changed all the code to remove the unused `auto_create` parameter.

Reply:
ok